src/onegov/org/models/traitinfo.py

```diff
@@ -134,7 +135,7 @@ def get_form_class(
     def get_editbar_links(
         self,
         request: 'OrgRequest'
-    ) -> 'Sequence[Link | LinkGroup | IFrameLink]':
+    ) -> 'Sequence[Link | LinkGroup | BaseLink]':
```

Comment:
Could you change this to `BaseLink | LinkGroup` in the other pull request after merging with master?

`Link` is redundant because it is a subclass of `BaseLink`. Same thing for `get_edit_links`.